firefox_puppeteer/__init__.py

```diff
@@ -86,6 +86,13 @@ def security(self):
         See the :class:`~api.security.Security` reference.
         """
 
+    @use_class_as_property('api.utils.Utils')
+    def utils(self):
+        """Provides an api for interacting with utility actions.
+
+        See the :class:`~api.utils.Utils` reference.
+        """
+
```

Comment:
Please make sure to rebase your branch against master. It still contains changes which are already done by PR #100.